fairlearn/reductions/grid_search_classification.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import copy
+import numpy as np
+
+from fairlearn.metrics import DemographicParity
+
+
+class GridSearchClassification:
```

Comment:
This should inherit from a general class that describes the interface for fit, etc.

Reply:
We would have to define that, and while it might match the signature of things in sklearn etc. ….. In a strongly typed language, I'd agree. Do we gain anything from doing that in Python?